### PR TITLE
Uniform visual display of parameters (2nd part)

### PR DESCRIPTION
**Pull request by Tristan-WorkGH**
Next to #1423

### CODE REVIEW

File: src/components/dialogs/parameters/widget/ParameterLine.tsx [hunk 1]
```diff
@@ -186,3 +202,36 @@ const ParamLineSlider: FunctionComponent<
...
+
+    return (
+        /*<Grid item container spacing={1} padding={1}>
+            <Grid item xs={7}>
+                <Typography component="span" variant="body1">
+                    <Box fontWeight="fontWeightBold" m={1}>
+                        <FormattedMessage id={'FixedGenerators'} />
+                    </Box>
+                </Typography>
+            </Grid>*/
```
Comment (EstherDarkish) [inline src/components/dialogs/parameters/widget/ParameterLine.tsx:219]: Dead code

File: src/translations/extern-fr.json [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{
+}
```
[the remark this answers is not in the document]
Comment (Tristan-WorkGH) [reply src/translations/extern-fr.json:2]: no, will be another PR, as I haven't th translation from PO

File: src/components/dialogs/parameters/network-parameters.js [hunk 3]
```diff
@@ -29,13 +29,12 @@ export const FluxConventions = {
 
 export const NetworkParameters = ({ hideParameters }) => {
     const classes = useStyles();
-    const [fluxConventionLocal, handleChangeFluxConvention] = useParameterState(
+
+    const [, handleChangeFluxConvention] = useParameterState(
```
Comment (Walid-Sahnoun) [inline src/components/dialogs/parameters/network-parameters.js:33]: remove ,

File: src/components/dialogs/parameters/widget/ParameterGroup.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+                    TransitionProps={{
+                        unmountOnExit: props.unmountOnExit || false,
+                    }}
+                    disabled={props.disabled || undefined}
+                >
+                    <AccordionSummary
+                        className={classes.subgroupParametersAccordionSummary}
+                        expandIcon={
+                            mouseHover ? <ExpandCircleDown /> : <ExpandMore />
+                            /*<ArrowForwardIosSharp sx={{ fontSize: '0.9rem' }} />*/
```
Comment (Walid-Sahnoun) [inline src/components/dialogs/parameters/widget/ParameterGroup.tsx:58]: remove commented code

File: src/components/dialogs/parameters/widget/ParameterGroup.tsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+                        props.infoText && (
+                            <Typography
+                                sx={{ color: 'text.secondary' }}
+                                noWrap={true}
+                                align="right"
+                                variant="body2"
+                            >
+                                {props.infoText}
+                            </Typography>
+                        )*/}
```
Comment (Walid-Sahnoun) [inline src/components/dialogs/parameters/widget/ParameterGroup.tsx:77]: also here

File: src/translations/fr.json [hunk 6]
```diff
@@ -269,7 +269,7 @@
     "sensitivityAnalysisError": "Une erreur est survenue lors de l'exécution de l'analyse de sensibilité",
     "SensitivityInDeltaMW": "Sensibilité au niveau des quadripôles en ΔMW",
     "SensitivityInDeltaA": "Sensibilité au niveau des quadripôles en ΔA",
-    "SensitivityAtNode": "Sensibilité au niveau des noeuds",
+    "SensitivityAtNode": "Sensibilité au niveau des nœuds",
```
Comment (Walid-Sahnoun) [inline src/translations/fr.json:272]: why this change ?

File: src/translations/fr.json [hunk 7]
```diff
@@ -603,7 +603,7 @@
     "Local": "Local",
     "Distant": "Distant",
     "VoltageText": "Tension",
-    "QPercentText": "Coefficient de participation",
+    "QPercentText": "Cœfficient de participation",
```
Comment (Walid-Sahnoun) [inline src/translations/fr.json:606]: why this change ?

File: src/translations/fr.json (hunk 6 above)
Comment (Tristan-WorkGH) [reply src/translations/fr.json:272]: [linguistic rule](https://fr.wikipedia.org/wiki/%C5%92#Emplois_modernes) in french

File: src/components/dialogs/parameters/network-parameters.js (hunk 3 above)
Comment (Tristan-WorkGH) [reply src/components/dialogs/parameters/network-parameters.js:33]: I can't as I want to get the second object in the array returned by the function, not the first

**Review by EstherDarkish — APPROVED** [review]
Code OK, tests OK

**Review by Walid-Sahnoun — APPROVED** [review]
code ok
test ok
